src/pages/index.tsx

```diff
@@ -1,15 +1,7 @@
+/* eslint-disable react/jsx-key */
```

Comment:
We shouldn't disable this rule, we need the keys to be set so that React can optimize our lists.
```suggestion
```